docs/userguide.rst

```diff
@@ -315,6 +315,33 @@ and push the created image to the DockerHub image registry:
 
     $ docker push johndoe/myenv
 
+**Supporting non-root users**
+
+Running containers with root user is not secure and therefore dicouraged. If
+you want to improve the security in your environment you can setup your own
+user.
+
+In order to be compatible with the REANA platform you should do the following:
+
+- Add your user to the root group, so anything created by this user will be
+  accessible by the REANA user, who also belongs to the root group.
+
+- Any directory or files used in your analysis should have the correct
```

Comment:
I think people won't understand what they need to do for "should have correct permissions"... We can leave this bit out, since usually they don't need to do anything and/or if they want to force a certain UID then they will most probably set the permissions right naturally...